omnibus/files/private-chef-cookbooks/private-chef/templates/default/sv-rabbitmq-check.erb

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/usr/bin/env ERL_EPMD_ADDRESS=127.0.0.1 HOME=/root /opt/opscode/embedded/bin/rabbitmqctl wait "<%= node['private_chef']['rabbitmq']['data_dir'] %>/<%= node['private_chef']['rabbitmq']['nodename'] %>.pid"
```

Comment:
@marcparadise This is some magic of the runit cookkbook, like the other sv- templates.